grpc/client/src/main/java/io/helidon/grpc/client/GrpcServiceClient.java

```diff
@@ -139,6 +136,27 @@ public String serviceName() {
         return observer.getFuture();
     }
 
+    /**
+     * Invoke a unary method that doesn't take any request parameter.
+     * @param methodName Method name
+     * @param <RespT> The type of response.
+     * @return The response.
+     */
+    public <RespT> CompletableFuture<RespT> unary(String methodName) {
+        //TODO: Check if the following is OK.
+        return unary(methodName, "");
+    }
+
+    /**
+     * Invoke a unary method that doesn't take any request parameter.
+     * @param methodName Method name
+     * @param <RespT> The type of response.
+     * @return The response.
+     */
+    public <RespT> CompletableFuture<RespT> blockingUnary(String methodName) {
+        return blockingUnary(methodName, "");
```

Comment:
Same comment about passing `""`

Reply:
For the above two comments, here is the plan:

1. I think we have to support the above API (`blockingUnary(String methodName)`) as the "minimalist gRPC application" in chapter one doesn't take any arg. So, it is fair that we support this aPI.

2. Now regarding the use of `io.helidon.grpc.core.proto.Types.Empty.getDefaultInstance()` I will have to wait till the `grpc-mp` branch is merged. The `proto` package is not in the core module yet.

Reply:
You cannot have a method that does not work (and hence cannot possibly have been properly tested) just because you have written an example in the docs that uses it. I'd argue that your documentation is wrong in this case and you should change it to be making a proper unary call with a request and response. Right now the client can only support proper unary method signatures, the same goes for all of the other method types. Support for other signatures is handled in gRPC MP but under the covers they still end up as proper unary, server streaming or whatever calls by the time they call methods on `GrpcServiceClient`. I really do not think that the `GrpcServiceClient ` class needs to support anything but the standard calls.

Reply:
I agree with JK -- I don't think we need or want the SE client to support this. The server certainly doesn't.

I'm ok with supporting it in the MP server and client, as that's where we do the magic of passing `Empty` as argument or return value as necessary, but the SE client and server are simply exposing gRPC as-is, in a slightly friendlier and more Helidon-like form.

Let's get rid of these methods and only expose methods that actually work without any "magic" on our part.

Reply:
Done. Remove those two methods.